### PR TITLE
components/routes: add Routes route and core components for List view

### PR DESCRIPTION
**Pull request by maceksimon**
First step in implementing the "Routes" section of the app:

1. Create a `/routes` url where components will be displayed. (Direct sidebar navigation to that route.)

2. Add a set of components displaying a List view:

* `RouteListDisplay`
* `RouteItemDisplay`
* `RouteListEdit`
* `RouteItemEdit`

Components are available for preview in component test suite.
Fixtures `routeList` and `routeListItem` represent examples of the current data structure for a route list.

### CODE REVIEW

File: src/components/__tests__/RouteItemEdit.cy.js [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+
+  context('mobile', () => {
+    beforeEach(() => {
+      cy.fixture('routeListItem').then((routes) => {
+        cy.mount(RouteItemEdit, {
+          props: {
+            route: routes.toWork,
+          },
+        });
+        cy.viewport('iphone-6');
```
Comment (tmszi) [inline src/components/__tests__/RouteItemEdit.cy.js:117]: Do you test how is layout rendered (mobile viewport vs desktop viewport)?

File: src/components/routes/RouteItemEdit.vue [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+      <!-- Column: Direction -->
+      <div
+        class="flex gap-8 text-subtitle2 text-weight-bold text-grey-10"
+        data-cy="label-direction"
+      >
+        <!-- From work -->
+        <span v-if="route.direction === 'from_work'">
+          <q-icon
+            name="arrow_back"
+            size="18px"
```
Comment (tmszi) [inline src/components/routes/RouteItemEdit.vue:112]: Size prop value should be const (value repeat on the code line 121).

File: src/i18n/cs.toml [hunk 3]
```diff
@@ -376,3 +379,17 @@ messagePasswordConfirmNotMatch = "Hesla se neshodují"
...
+actionInputDistance = "Zadat vzdálenost"
+labelDistance = "Trasa"
+labelDirectionFromWork = "Z práce"
+labelDirectionToWork = "Do práce"
+labelTransportType = "Typ dopravy"
+
+[routes.transport]
+bike = "Na kole"
+car = "Autem"
+none = "Nijak"
```
Comment (tmszi) [inline src/i18n/cs.toml:393]: Small note maybe better translation would be _žádný_.

File: src/i18n/en.toml [hunk 4]
```diff
@@ -21,6 +21,7 @@ female = "Female"
 man = "Man"
 male = "Male"
 woman = "Woman"
+km = "km"
```
Comment (tmszi) [inline src/i18n/en.toml:24]: better var name `routeLengthUnit = "km"`

File: src/components/__tests__/RouteItemEdit.cy.js (hunk 1 above)
Comment (maceksimon) [reply src/components/__tests__/RouteItemEdit.cy.js:117]: Did not test this before. Added.

File: src/i18n/en.toml (hunk 4 above)
Comment (maceksimon) [reply src/i18n/en.toml]: Updated.

File: src/i18n/cs.toml (hunk 3 above)
Comment (maceksimon) [reply src/i18n/cs.toml]: Good point. Added "žádná" as in "žádná cesta".

File: src/components/routes/RouteItemEdit.vue (hunk 2 above)
Comment (maceksimon) [reply src/components/routes/RouteItemEdit.vue]: Refactored into a variable.

**Review by tmszi — APPROVED** [review]

**Pull request merged**